### ISSUE TITLE
As a reviewer, I can see and review bookings

### ISSUE THREAD

**Issue by Alex-Hayeur** [CSU-Booking-Platform/application#24]
### User story
As an approver, I need to be able to approve or deny bookings for rooms I manage

### Narrative
When a new room booking is made, the system will require approval from the managers of that room before confirming the booking. For example, Conference Room bookings require approval from the Secretaries, and Lounge bookings require approval of Security.

### Out of scope
This issue is separate from events that would automatically make a room non-bookable. It is only related to administrators being notified that they should manually approve or deny an existing booking.

### UI Mockups

![#295-1](https://user-images.githubusercontent.com/31771444/104143668-69c65880-538e-11eb-8212-e4e884417975.PNG)
![#295-2](https://user-images.githubusercontent.com/31771444/104143672-6b901c00-538e-11eb-882f-10f302b7d563.PNG)

### Acceptance criteria

| # | Given | When | Then |
|---|-------|------|------|
|1|I am logged in as a manager for a room|There is a booking pending review |I am given an option to approve or deny that booking|
|2|I am logged in as a manager for a room|I navigate to the bookings list |I am given a list of bookings that are awaiting approval|

**Comment by alexstojda** [comment]
Before we can start this story, we will need to learn the full approval process & steps in order to effectively replicate this process in the system. 

Moving to Icebox until the above information has been clarified.

**Comment by EvanDime** [comment]
For now, this issue will need to list all pending booking requests. the review should be able to click on one. And form there approve or deny it.
